RELEASE.md

```diff
@@ -68,18 +105,21 @@ Check that the released files, library and symbols, are available at the followi
     - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/mongoodbc.msi`
 - Ubuntu 2204
   - Release build
-    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/${release_version}/release/libatsql.so`  
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/${release_version}/release/libatsql.so`
     - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/${release_version}/release/mongoodbc.tar.gz`
 
 ##### Verify that the driver works with PowerBI
+
 Download and install the driver file.
 
 Verify that it is able to connect to Atlas Data Federation with PowerBI, extract data,
 and add columns to the worksheet.
 
 #### Close Release Ticket
+
 Move the JIRA ticket tracking this release to the "Closed" state.
 
 #### Ensure next release ticket and fixVersion created
-Ensure that a JIRA ticket tracking the next release has been created
-and is assigned the appropriate fixVersion.
+
+Ensure that a JIRA ticket tracking the next release has been created and is assigned the appropriate fixVersion. The fixVersion should
```

Comment:
[nit]
There will be 2 fix versions in that field for driver releases now. That's also what you have in mind right?

Reply:
Well, I was envisioning the entire fixVersion being what the tag should include, similar to https://jira.mongodb.org/browse/SQL-2439

Reply:
I was thinking about this field :
Fix Version/s: [odbc-2.0.0-alpha](https://jira.mongodb.org/issues/?jql=project+%3D+SQL+AND+fixVersion+%3D+odbc-2.0.0-alpha)
and adding libmongosqltranslate version to that list.

Reply:
I like that. It allows more flexibility.